spec/fake_app.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```

Comment:
Using an approach from Kaminari to get away with being able to test things but not having to involve engine_cart or a whole test app.

Reply:
![neat!](https://media.tenor.com/images/f83e5060dde1137714c2b93bb30fd491/tenor.gif)